src/algebra/big_operators/fin.lean

```diff
@@ -64,10 +64,7 @@ is the sum of `f x`, for some `x : fin (n + 1)` plus the remaining product -/]
 theorem prod_univ_succ_above [comm_monoid β] {n : ℕ} (f : fin (n + 1) → β)
   (x : fin (n + 1)) :
   ∏ i, f i = f x * ∏ i : fin n, f (x.succ_above i) :=
-begin
-  rw [fintype.prod_eq_mul_prod_compl x, ← image_succ_above_univ, prod_image],
-  exact λ _ _ _ _ h, x.succ_above.injective h
-end
+by { rw [univ_succ_above, prod_cons, finset.prod_map], refl }
```

Comment:
What's the goal at the `rfl`? Are we missing a lemma?

Reply:
No we have it, but under a nonsexy name. `function.embedding.coe_fn_mk`